stable/admin/templates/_sidecar.tpl

```diff
@@ -0,0 +1,86 @@
+{{/*
+Return the proper nuocollector image name
+*/}}
+{{- define "nuocollector.image" -}}
+{{- $registryName := $.Values.nuocollector.image.registry -}}
+{{- $repositoryName := $.Values.nuocollector.image.repository -}}
+{{- $tag := $.Values.nuocollector.image.tag | toString -}}
+{{/*
+Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
+but Helm 2.9 and 2.10 doesn't support it, so we need to implement this if-else logic.
+Also, we can't use a single if because lazy evaluation is not an option
+*/}}
+{{- if .Values.global }}
+    {{- if .Values.global.imageRegistry }}
+        {{- printf "%s/%s:%s" .Values.global.imageRegistry $repositoryName $tag -}}
+    {{- else -}}
+        {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+    {{- end -}}
+{{- else -}}
+    {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper nuocollector config watcher image name
+*/}}
+{{- define "nuocollector.watcher" -}}
+{{- $registryName := .Values.nuocollector.watcher.registry -}}
+{{- $repositoryName := .Values.nuocollector.watcher.repository -}}
+{{- $tag := .Values.nuocollector.watcher.tag | toString -}}
+{{/*
+Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
+but Helm 2.9 and 2.10 doesn't support it, so we need to implement this if-else logic.
+Also, we can't use a single if because lazy evaluation is not an option
+*/}}
+{{- if .Values.global }}
+    {{- if .Values.global.imageRegistry }}
+        {{- printf "%s/%s:%s" .Values.global.imageRegistry $repositoryName $tag -}}
+    {{- else -}}
+        {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+    {{- end -}}
+{{- else -}}
+    {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "nuodb.sidecar" -}}
+{{- if $.Values.nuocollector }}
+{{- if $.Values.nuocollector.enabled }}
+- name: nuocollector
+  image: {{ template "nuocollector.image" . }}
+  imagePullPolicy: {{ $.Values.nuocollector.image.pullPolicy }}
+  tty: true
+  volumeMounts:
+  - mountPath: /etc/telegraf/telegraf.d/dynamic/
+    name: nuocollector-config
+  - name: log-volume
+    mountPath: /var/log/nuodb
+- name: nuocollector-config
+  image: {{ template "nuocollector.watcher" . }}
+  imagePullPolicy: {{ $.Values.nuocollector.watcher.pullPolicy }}
+  env:
+  - name: LABEL
+    value: "nuodb.com/nuocollector-plugin in ({{ template "admin.fullname" $ }}, insights)"
+  - name: FOLDER
```

Comment:
We might want to have an optional env variable LABEL_VALUE. and allow different labels for different databases or domains.   Probably not an issue at moment.  we'll see.